src/WebJobs.Extensions.DurableTask/LocalGrpcListener.cs

```diff
@@ -195,6 +226,64 @@ public override Task<Empty> Hello(Empty request, ServerCallContext context)
                 }
             }
 
+            private DurabilityProvider GetDurabilityProviderToScheduleOrchestration(ServerCallContext context)
```

Comment:
Is this new method needed? Could we instead use the existing `GetClient(context)` method?

Reply:
I replaced it with a call to the existing `GetDurabilityProvider(context)` method.